packages/scipy/patches/0002-Fix-getbreak-dlamch.patch

```diff
@@ -0,0 +1,39 @@
+From dfad8166e0705c6808035b67b761d27bb0649ec4 Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Fri, 18 Mar 2022 16:06:48 -0700
+Subject: [PATCH 02/15] Fix getbreak dlamch
+
+---
+ scipy/sparse/linalg/_isolve/iterative/GMRESREVCOM.f.src | 2 +-
+ scipy/sparse/linalg/_isolve/iterative/getbreak.f.src    | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/scipy/sparse/linalg/_isolve/iterative/GMRESREVCOM.f.src b/scipy/sparse/linalg/_isolve/iterative/GMRESREVCOM.f.src
+index da4bde8ec..5f47afde2 100644
+--- a/scipy/sparse/linalg/_isolve/iterative/GMRESREVCOM.f.src
++++ b/scipy/sparse/linalg/_isolve/iterative/GMRESREVCOM.f.src
+@@ -154,7 +154,7 @@
+       INFO = 0
+       MAXIT = ITER
+       BRKDWN = .FALSE.
+-      EPS = <sdsd>LAMCH('EPS')
++      EPS = <sdsd>LAMCH(69)
```

Comment:
This looks strange to me. Why 69? LAMCH('EPS') is supposed to return machine precision if I understand correctly:
 - http://netlib.org/lapack/explore-html-3.3.0-2011-01-25/dlamch_8f_source.html
 - http://www.icl.utk.edu/~luszczek/lapack/lamch.html

Reply:
69 is the ascii code for "E".

Reply:
OK, that makes sense thanks. Maybe worth adding in the patch comment.

Reply:
But I think this patch isn't needed it should get handled by `char1_args_to_int`.